src/ciceroscm/concentrations_emissions_handler.py

```diff
@@ -186,10 +188,15 @@ def __init__(
         self.forc = {}
         self.nat_emis_ch4 = input_handler.get_data("nat_ch4")
         self.nat_emis_n2o = input_handler.get_data("nat_n2o")
+        print("Carbon Model=" + pamset["carbon_cycle_model"])
         self.pamset = cut_and_check_pamset(
             {"idtm": 24, "nystart": 1750, "nyend": 2100, "emstart": 1850},
             pamset,
-            used={"rs_function": "missing", "rb_function": "missing"},
+            used={
+                "rs_function": "missing",
+                "rb_function": "missing",
+                "carbon_cycle_model": "default",
+            },
             cut_warnings=True,
         )
```

Comment:
Unless you add more manualy to the used, or change the cut_and_check_pamset definition, you will toss out parameters here that the carbon_cycle_model might be interested in later, maybe the used needs to have some info from the carbon_cycle model itself can provide?